lessons/session_3/session_3_training.md

```diff
@@ -0,0 +1,35 @@
+---
+marp: true
+theme: uncover
+_class: invert
+paginate: true
+output: pdf_document
+---
+
+# ML Systems in Production
+# Session 3: Training
+Jesús A. Calderón
+February 2024
+
+---
+
+# Agenda
+
+**1.1 Overview of ML Systems**
```

Comment:
I'm assuming this should be 3.1 and not 1.1.

Reply:
.